### PR TITLE
[FEATURE] Build formula with Ritchie-cli

### PR DESCRIPTION
**Pull request by kaduartur**
**- What I did**
I created a new command `rit build formulas` to build formulas locally.
rit-build-formula

**- How to verify it**
- Run the command `rit build formulas` to build formulas, you can run the command with the --watch flag `rit build formulas --watch` and get real-time updates.
- Run tests with the command `make unit-test`

**- Description for the changelog**
Created the build.go and watcher.go to build and watch your formulas locally.

### CODE REVIEW

File: pkg/formula/build/build.go [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+	if err := b.copyConfig(formulaPath, formulaDestPath); err != nil {
+		return nil, err
+	}
+
+	if err := b.copyTree(workspacePath); err != nil {
+		return nil, err
+	}
+
+	return nil, nil
```
Comment (sandokandias) [inline pkg/formula/build/build.go:60]: Remove `[]byte` from `func` return

**Comment by kaduartur** [comment]
[[FEATURE UX] rit build formula](https://trello.com/c/wepGioHa/433-feature-ux-rit-build-formula)

**Comment by sandokandias** [comment]
very good!

**Pull request merged**